pyuvsim/antenna.py

```diff
@@ -47,7 +47,9 @@ def get_beam_jones(self, array, source_alt_az, frequency, reuse_spline=True,
         # 2x2 array of Efield vectors in alt/az
 
         # convert to UVBeam az/za convention
-        source_za, source_az = simutils.altaz_to_zenithangle_azimuth(source_alt_az[0], source_alt_az[1])
+        source_za, source_az = simutils.altaz_to_zenithangle_azimuth(
+            source_alt_az[0], source_alt_az[1]
+        )
```

Comment:
I've seen this approach before and I get that it reduces the number of line changes in diffs. But I don't personally love it. I'm also not sure it's what PEP8 asks for, but I guess it's passing that check.

Reply:
I personally much prefer it. It has the benefit you mentioned, but also means that long lists of arguments are all indented at the same level for all functions called at that level, and also minimizes number of lines used (while staying under a line limit). It's how `black` does it (except it also does a thing which I find a bit annoying: it puts _every_ argument on a new line, if the arguments don't all fit on one, which takes too many lines sometimes).

Reply:
I guess the thing that bothers me most is the parenthesis on its own line.

Reply:
I like it -- it gives a simple visual clue as to when a function call has ended. I'm happy to move these to the end of the previous line if you/others feel strongly. I don't think there's any consensus in PEP8 or otherwise on this.

In https://community.lsst.org/t/pep8-style-hanging-indentation-and-closing-brackets/618 most people seem to like my style, though they agree that aligning with opening bracket is good too.
This post: https://treyhunner.com/2017/07/craft-your-python-like-poetry/ also seems to prefer this method.

Reply:
it's ok, I think I'm in the minority on this.